src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -364,6 +364,10 @@
     "NAME": "Materialize",
     "DESCRIPTION": "Gain !B! Block. NL Choose 1 of 3 stsjorbsmod:Material Components."
   },
+  "stsjorbsmod:Melt": {
+    "NAME": "Melt",
+    "DESCRIPTION": "Apply burning equal to target's block. NL Deal !D! damage."
```

Comment:
`Block` is usually capitalized in card descriptions (I don't know why the base game capitalizes `Block` but not `damage`...)